app/src/components/mode-specific/desktop/InterceptTraffic/WebTraffic/TrafficTableV2/FixedRequestLogPane.css

```diff
@@ -116,3 +116,12 @@
   width: 220px;
   margin-top: -10px;
 }
+
+[data-reach-tab-panels] > div {
+  min-height: 180px;
+}
+.navigation-panel-wrapper {
+  overflow-y: "auto";
+  height: "100%";
+  padding: "0.8rem";
```

Comment:
In CSS file, you don't use double quotes. Also, try to use pixel unit instead of rem.